model_inspector/get_inspector.py

```diff
@@ -60,7 +62,7 @@ def get_inspector(model: BaseEstimator, X: pd.DataFrame, y: pd.Series) -> _Inspe
         return _SearchCVInspector(model, X, y)
     # `type_of_target` can't reliably distinguish between continuous and
     # multiclass
-    if isinstance(model, RegressorMixin):
+    if isinstance(model, (RegressorMixin, CatBoostRegressor)):
```

Comment:
`CatBoostRegressor` doesn't inherit from `RegressorMixin`, so without this change Model Inspector thinks it is a multiclass model

Reply:
Since we require a fitted model, should we check that is is actually fitted?

Reply:
https://scikit-learn.org/stable/modules/generated/sklearn.utils.validation.check_is_fitted.html

Reply:
Never mind, you are doing it in the inspector itself!